marketing_transactional.go

```diff
@@ -0,0 +1,50 @@
+package hubspot
+
+import "fmt"
+
+const transactionalBasePath = "transactional"
+
+// TransactionalService is an interface for the marketing/transactional service of the HubSpot API.
+// Reference: https://developers.hubspot.com/docs/api/marketing/transactional-emails
+type TransactionalService interface {
+	SendSingleEmail(props *SendSingleEmailProperties) (*SendSingleEmailResponse, error)
+}
+
+// Provides the default implementation of TransactionService.
+type TransactionalServiceOp struct {
+	transactionalPath string
+	client            *Client
+}
+
+var _ TransactionalService = (*TransactionalServiceOp)(nil)
+
+type SendSingleEmailMessage struct {
+	To      string   `json:"to"`
+	From    string   `json:"from,omitempty"`
+	SendId  string   `json:"sendId,omitempty"`
+	ReplyTo []string `json:"replyTo,omitempty"`
+	Cc      string   `json:"cc,omitempty"`
```

Comment:
From following the api document, `Cc` and `Bcc` are array types.
https://developers.hubspot.com/docs/api/marketing/transactional-emails#endpoint?spec=POST-/marketing/v3/transactional/single-email/send
```suggestion
	Cc      []string   `json:"cc,omitempty"`
	Bcc     []string   `json:"bcc,omitempty"`
```